data/functions/clean.sql

```diff
@@ -8,6 +8,16 @@ begin
 end
 $$ language plpgsql;
 
+
+-- There are weird explainations in the dimpreport table that appear in curly braces
```

Comment:
* explainations -> explanations
* dimpreport -> dimreporttype

How about a more specific description of the problem this function is solving? I'm not sure that values of `rpt_tp_desc` are weird--it's just that they sometimes include additional descriptions in curly braces that we don't want to show. An example might help.